react/features/calendar-sync/components/ConferenceNotification.native.js

```diff
@@ -0,0 +1,242 @@
+                                    {
+                                        getLocalizedDateFormatter(
+                                            event.startDate
+                                        ).fromNow()
+                                    }
+                                </Text>
+                            </View>
+                            <View style = { styles.notificationIconContainer }>
+                                <Icon
+                                    name = 'navigate_next'
+                                    style = { styles.notificationIcon } />
+                            </View>
+                        </View>
+                    </TouchableOpacity>
+                </View>
+            );
+        }
+
+        return null;
+    }
+
+    _getContainerStyle: () => Array<Object>;
+
+    /**
+     * Assembles a style array for the container.
+     *
+     * @private
+     * @returns {Array<Object>}
+     */
+    _getContainerStyle() {
+        const { event } = this.state;
+        const now = Date.now();
+        const style = [
+            styles.notificationContainer,
+            {
+                top: this.props._screenHeight / 10
+            }
+        ];
+
+        if (event && event.startDate < now && event.endDate > now) {
+            style.push(styles.notificationContainerPast);
+        }
+
+        return style;
+    }
+
+    _maybeDisplayNotification: () => void;
+
+    /**
+     * Periodically checks if there is an event in the calendar for which we
+     * need to show a notification.
+     *
+     * @private
+     * @returns {void}
+     */
+    _maybeDisplayNotification() {
+        const { _currentConferenceURL, _eventList } = this.props;
+        let eventToShow;
+
+        if (_eventList && _eventList.length) {
+            const now = Date.now();
+
+            for (const event of _eventList) {
+                if (event.url !== _currentConferenceURL) {
+                    if ((!eventToShow
+                        && event.startDate > now
+                        && event.startDate < now + ALERT_MILLISECONDS)
+                        || (event.startDate < now && event.endDate > now)
+                    ) {
+                        eventToShow = event;
+                    }
+                }
+            }
+        }
+
+        this.setState({
+            event: eventToShow
+        });
+    }
+
+    _onGoToNext: () => void;
+
+    /**
+     * Opens the meeting URL that the notification shows.
+     *
+     * @private
+     * @param {string} url - The URL to open.
+     * @returns {void}
+     */
+    _onGoToNext() {
+        const { event } = this.state;
+
+        if (event && event.url) {
+            this.props.dispatch(appNavigate(event.url));
+        }
+    }
+
+}
+
+/**
+ * Maps redux state to component props.
+ *
+ * @param {Object} state - The redux state.
+ * @returns {{
+ *      _eventList: Array,
+ *      _screenHeight: number
```

Comment:
Missing _currentConferenceURL in this jsdoc. I tend to miss return values in mapStateToProps all the time, which maybe is a sign of something else.